### PR TITLE
Add apache cordova cheat sheet

### PR DESCRIPTION
**Pull request by marti1125**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [X] Cheat Sheets: **`New Apache Cordova Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Apache Cordova Cheat Sheet

https://duck.co/ia/view/apache_cordova_cheat_sheet

---
Instant Answer Page: https://duck.co/ia/view/apache_cordova_cheat_sheet
Maintainer: @mention
@marti1125
marti1125@gmail.com

### CODE REVIEW

**Comment by daxtheduck** [comment]
@marti1125 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by marti1125** [comment]
Yeah!! pass the tests =D

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=apache cordova cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/apache_cordova_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=apache cordova cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/apache_cordova_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=apache cordova cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/apache_cordova_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=apache cordova cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/apache_cordova_mobile.png)

**Comment by tagawa** [comment]
Hi @marti1125! Thank you for creating this cheat sheet. Good to see that the checks passed and screenshot's OK. As you may have seen, it's visible on our beta server: https://beta.duckduckgo.com/?q=apache+cordova+cheat+sheet&ia=cheatsheetscontroller
so it can be tested and hopefully merged soon.

**Comment by marti1125** [comment]
@tagawa Thanks!! I will make more =D

File: share/goodie/cheat_sheets/json/apache-cordova.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
+{
+    "id": "apache_cordova_cheat_sheet",
+    "name": "Apache Cordova",
+    "description": "Apache Cordova is a popular mobile application development framework originally created by Nitobi.",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/apache-cordova.json]: Lets remove period at end of desc

Comment (marti1125) [inline share/goodie/cheat_sheets/json/apache-cordova.json]: Could I to change it? update PR =?

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/apache-cordova.json]: @marti1125 If you push this branch it will update the PR.

**Comment by marti1125** [comment]
@jophab @GuiltyDolphin done :D

**Comment by marti1125** [comment]
in "val": "Before running this command, you need to set up the device for testing, following procedures that vary for each platform." Is it correct =?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=apache cordova cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/apache_cordova_mobile.png?3BtjCWU0Yo)

**Comment by tagawa** [comment]
It looks like the Travis checks were failing because the IA page ID didn't match. I've udpated it so hopefully it's OK now: https://duck.co/ia/view/apache_cordova_cheat_sheet

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=apache cordova cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/apache_cordova_cheat_sheet_mobile.png?TD0Mjh2FJI)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=apache cordova cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/apache_cordova_cheat_sheet_mobile.png?ZakKwmFN2Z)

**Comment by mintsoft** [comment]
@tagawa @marti1125 I've just repushed the build so this should now be green :+1:

**Comment by tagawa** [comment]
And indeed it is! Thanks @mintsoft

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=apache cordova cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/apache_cordova_cheat_sheet_mobile.png?x44VJL5DCY)

**Comment by tagawa** [comment]
Hi @marti1125. Sorry for the long delay in this. It all looks good and there have been no further comments so let's merge and it should go live in a few days.

Thanks again for creating this for Cordova users, and for your patience.

**Comment by daxtheduck** [comment]
@marti1125 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by marti1125** [comment]
GREAT!!!

**Comment by tagawa** [comment]
Aaaaand this is live already :-) https://duckduckgo.com/?q=apache+cordova+cheat+sheet

**Pull request merged**